src/lookup.cpp

```diff
@@ -50,23 +54,31 @@ static const struct option LONG_OPTS[] = {
 };
 
 void usage(){
-    cout << "       *** " << PROGRAM_NAME << " " << VERSION_LONG << " ***       \n"
-            "Usage:                                                             \n"
-            "   cnf-lookup [ -d ] <search term>                                 \n"
-            "                                                                   \n"
-            "Options:                                                           \n"
-            " --help            -? -h     Show this help and exit               \n"
-            " --verbose         -v        Display verbose output                \n"
-            "                                                                   \n"
-            " --database-path   -d        Customize the database lookup path    \n"
-            "                             default is " << DATABASE_PATH << "    \n"
-            " --colors          -c        Pretty colored output                 \n"
+    cout << format(bl::translate("       *** %s %s ***\n"))
```

Comment:
please make sure this block remains 'readable'. right now indentations and closing braces are quite messed up.